src/Javis.jl

```diff
@@ -275,15 +276,35 @@ function render(
         tempdirectory = mktempdir()
     end
 
+    unprocessed_frames = Vector{Matrix{RGB{Normed{UInt8,8}}}}()
+
     filecounter = 1
-    @showprogress 1 "Rendering frames..." for frame in frames
+    @showprogress 1 "Drawing frames..." for frame in frames
         frame_image = convert.(RGB, get_javis_frame(video, objects, frame; layers = layers))
+        push!(unprocessed_frames, frame_image)
```

Comment:
I think saving them all into a vector might slow down the render function dramatically.
I'm wondering whether it makes more sense to call `postprocess` with the name of the tempdirectory folder instead. 
That would only work for `gif` though. Another option would maybe be to check if the `postprocess` function is the identity function. If that is the case then do nothing otherwise use the workflow you provided. However for simple postprocessing where one does a change for each frame and don't need all of them this current way would still potentially slow it down/fill up the memory.

Reply:
Ok I thought the vector would be a catch but don't really know how to work around that. Let me try a bit more if I don't come up with anything I add it only as a frame by frame function and then for the reversing I add it as a special case with its own keyword argument maybe?

Reply:
Yeah the problem is that all the images need to be in memory for this. Which is probably fine for small short animations but for people who want to create YT videos with it it's probably not going to end up well :smile: 
Special case for reverse seems reasonable to me.